src/common/tvgCompressor.cpp

```diff
@@ -444,6 +444,7 @@ size_t b64Decode(const char* encoded, const size_t len, char** decoded)
     auto reserved = 3 * (1 + (len >> 2)) + 1;
     auto output = static_cast<char*>(malloc(reserved * sizeof(char)));
```

Comment:
1. please use `calloc()`
2. Shouldn't _simpleXmlSkipWhiteSpace stop the iteration when the string ends `\0`?

Reply:
1. Changed.
2. As far as I know, `itrEnd` of `_simpleXmlSkipWhiteSpace` is set by strlen(). So, the end of `itrEnd` is probably `\0`. Is there any problem or suspicious point?